io_scene_nif/modules/property/material/material_export.py

```diff
@@ -59,8 +59,7 @@ def export_material_property(self, name, flags, ambient, diffuse, specular, emis
 
         # list which determines whether the material name is relevant or not  only for particular names this holds,
```

Comment:
todo: extract to function

Reply:
Yeah no idea why i formatted this. Yeah export code needs same TLC as import side